velafrica/core/settings.py

```diff
@@ -27,13 +27,7 @@
 SECRET_KEY = os.environ['SECRET_KEY']
 
 # SECURITY WARNING: don't run with debug turned on in production!
-if 'DEBUG' in os.environ:
-    if os.environ['DEBUG'] == 'True':
-        DEBUG = True
-    else:
-        DEBUG = False
-else:
-    DEBUG = False
+DEBUG = True if 'DEBUG' in os.environ and os.environ['DEBUG'] == 'True' else False
```

Comment:
nice, sollte sogar noch einfacher gehen:

DEBUG = 'DEBUG' in os.environ and os.environ['DEBUG'] == 'True'

Reply:
Gemacht siehe [1459c60](https://github.com/velafrica/velafrica-django/pull/4/commits/1459c6092cb72f9b6c99a109b8890b9b1cb0d006)